posthog/api/organization.py

```diff
@@ -118,10 +119,11 @@ def get_teams(self, instance: Organization) -> List[Dict[str, Any]]:
         visible_teams = [team for team in teams if team["effective_membership_level"] is not None]
         return visible_teams
 
-    def get_metadata(self, instance: Organization) -> Dict[str, int]:
+    def get_metadata(self, instance: Organization) -> Dict[str, Any]:
```

Comment:
Much better than Any, since it's more precise

```suggestion
    def get_metadata(self, instance: Organization) -> Dict[str, Union[int,str]]:
```